x-pack/plugins/siem/public/pages/timelines/translations.ts

```diff
@@ -6,6 +6,10 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const ALL_TIMELINES = i18n.translate('xpack.siem.pages.timeline.allTimelinesTitle', {
+export const TIMELINES = i18n.translate('xpack.siem.timelines', {
```

Comment:
This key will need to end with a message type as per the [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#naming-convention).

Reply:
👍